content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +234,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
+Using the methods described above, customize your tracing configuration with these settings. Use the first name (for example, `DD_<INTEGRATION>_ENABLED`) when setting environment variables or configuration files. The second name (for example, `Enabled`), indicates the name for the `IntegrationSettings` property to use when changing settings in the code. Access these properties through the `TracerSettings.Integrations[]` indexer. 
 
-#### Unified Service Tagging
+### Unified Service Tagging
 
-| Setting Name                                        | Description                                                                                                                                                                                                       |
+To use [Unified Service Tagging][7], configure the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` settings for your services. 
+
+| Setting Name                                        |Description                                                                                                                                                                                                       |
 |-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `DD_ENV`<br/><br/>`Environment`                     | If specified, adds the `env` tag with the specified value to all generated spans. See [Agent configuration][8] for more details about the `env` tag. Available for versions 1.17.0+                                                           |
-| `DD_SERVICE`<br/><br/>`ServiceName`            | If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Available for versions 1.17.0+  |
-| `DD_VERSION`<br/><br/>`ServiceVersion`            | If specified, sets the version of the service. Available for versions 1.17.0+
-| `DD_TAGS`<br/><br/>`GlobalTags`       | If specified, adds all of the specified tags to all generated spans (e.g., `layer:api,team:intake`). Available for versions 1.17.0+                                                                                                                                              |
+| `DD_ENV`<br/><br/>`Environment`                     | If specified, adds the `env` tag with the specified value to all generated spans. Available for versions 1.17.0+                                           |
+| `DD_SERVICE`<br/><br/>`ServiceName`                 | If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Available for versions 1.17.0+      |
```

Comment:
This sounds fine, my only concern around this would be how we decide to word this with the limited space we have. If there are suggestions then please share - I am indifferent just want to call out the concerns.

Reply:
> with the limited space we have

My suggestion is shorter than the existing text 😅
Another even shorter alternative: `Added in x.y.z`.

Reply:
I like `Added in x.y.z` 😄